src/lib/tracker.js

```diff
@@ -0,0 +1,25 @@
+import Web from '@sap_oss/automated-usage-tracking-tool'
+
+const API_KEY = process.env.REACT_APP_RELEASE_BUILD ? process.env.REACT_APP_TRACKER_API_KEY_PROD : process.env.REACT_APP_TRACKER_API_KEY_DEV
+const DATA_CENTER = process.env.REACT_APP_RELEASE_BUILD ? process.env.REACT_APP_TRACKER_DATA_CENTER_PROD : process.env.REACT_APP_TRACKER_DATA_CENTER_DEV
+
+const trackingTool = new Web({
+  apiKey: API_KEY,
+  dataCenter: DATA_CENTER,
+})
+
+export async function requestConsentConfirmation() {
+  return await trackingTool.requestConsentConfirmation({
+    message: `
+      <h2>SAP Customer Data Cloud toolkit</h2>
+      This app collects anonymous usage data to help deliver and improve this product. By installing this app, you agree to share this information with SAP. If you wish to revoke your consent, please uninstall the app. Do you want to continue?
+    `,
+  })
+}
+
+export async function trackUsage({ featureName }) {
+  return await trackingTool.trackUsage({
+    toolName: 'sap-customer-data-cloud-toolkit',
+    featureName,
+  })
+}
```

Comment:
Currently, if there is no ".env" file or the tracking credentials are not there the `trackingTool` is still initialized and throws an error when calling the functions.

<img width="1719" alt="Screenshot 2024-09-05 at 14 40 31" src="https://github.com/user-attachments/assets/7860de24-22d1-4be3-9542-5efb9191cc04">

The app should still run even if the tracking is not initialized. But we can add a `console.log` message to notify that the tracking tool was not initialized.